src/modeler/AutoRest.Swagger.Tests/SwaggerModelerValidationTests.cs

```diff
@@ -14,552 +14,552 @@
+      messages.AssertOnlyValidationMessage(typeof(ResourceModelValidation), 1);
+    }
 
-        [Fact]
-        public void TrackedResourceGetOperationValidation()
-        {
-            var messages = ValidateSwagger(Path.Combine(Core.Utilities.Extensions.CodeBaseDirectory, "Resource", "Swagger", "Validation", "tracked-resource-get-operation.json"));
-            messages.AssertOnlyValidationMessage(typeof(TrackedResourceGetOperationValidation), 1);
-        }
+    [Fact]
+    public void SkuModelValidation()
+    {
+      var messages = ValidateSwagger(Path.Combine(Core.Utilities.Extensions.CodeBaseDirectory, "Resource", "Swagger", "Validation", "swagger-skumodel-validation.json"));
+      messages.AssertOnlyValidationMessage(typeof(SkuModelValidation), 1);
+    }
 
-        [Fact]
-        public void TrackedResource3Validation()
-        {
-            var messages = ValidateSwagger(Path.Combine(Core.Utilities.Extensions.CodeBaseDirectory, "Resource","Swagger", "Validation", "swagger-tracked-resource-3-validation.json"));
-            messages.AssertOnlyValidationMessage(typeof(TrackedResourceValidation), 1);
-        }
+    [Fact]
+    public void TrackedResource1Validation()
+    {
+      var messages = ValidateSwagger(Path.Combine(Core.Utilities.Extensions.CodeBaseDirectory, "Resource", "Swagger", "Validation", "swagger-tracked-resource-1-validation.json"));
+      messages.AssertOnlyValidationMessage(typeof(TrackedResourceValidation), 1);
+    }
 
-        [Fact]
-        public void TrackedResource4Validation()
-        {
-            var messages = ValidateSwagger(Path.Combine(Core.Utilities.Extensions.CodeBaseDirectory, "Resource","Swagger", "Validation", "swagger-tracked-resource-4-validation.json"));
-            messages.AssertOnlyValidationMessage(typeof(TrackedResourceValidation), 1);
-        }
+    [Fact]
+    public void TrackedResource2Validation()
+    {
+      var messages = ValidateSwagger(Path.Combine(Core.Utilities.Extensions.CodeBaseDirectory, "Resource", "Swagger", "Validation", "swagger-tracked-resource-2-validation.json"));
+      messages.AssertOnlyValidationMessage(typeof(TrackedResourceValidation), 1);
+    }
 
-        [Fact]
-        public void PutGetPatchResponseValidation()
-        {
-            var messages = ValidateSwagger(Path.Combine(Core.Utilities.Extensions.CodeBaseDirectory, "Resource","Swagger", "Validation", "swagger-putgetpatch-response-validation.json"));
-            messages.AssertOnlyValidationMessage(typeof(PutGetPatchResponseValidation), 1);
-        }
+    [Fact]
+    public void TrackedResourcePatchOperationValidation()
+    {
+      var messages = ValidateSwagger(Path.Combine(Core.Utilities.Extensions.CodeBaseDirectory, "Resource", "Swagger", "Validation", "tracked-resource-patch-operation.json"));
+      messages.AssertOnlyValidationMessage(typeof(TrackedResourcePatchOperationValidation), 1);
     }
 
-    #region Positive tests
+    [Fact]
```

Comment:
@olydis I'd like to remove .vscode from the repository then because that restricts me to usage of Visual Studio only but I like VS code :)

Do you have any concerns in removing the .vscode folder?